iot_relay_server/app/main.py

```
@@ -99,7 +152,65 @@ async def irrigation_events() -> list[dict]:
+    if not latest:
+        return {"error": "센서 데이터 없음"}
+    sensors_dict = {
+        "temperature": latest["temperature"],
+        "humidity": latest["humidity"],
+        "light_intensity": latest["lightIntensity"],
+        "soil_moisture": latest["soilMoisture"],
+    }
+    try:
+        decisions = await ai_agent.process_sensor_data(sensors_dict)
+        return {"decisions": decisions, "agent_enabled": ai_agent.agent_enabled}
+    except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)